### PR TITLE
Sync tests

### PR DESCRIPTION
**Pull request by czue**
don't merge, just running tests

### CODE REVIEW

**Comment by czue** [comment]
note to self:
```
case ids before update: 
dependent case ids before update: 
child_id-bdffda93c12a47b2ad553d69fd4e65ea: create
child_id-bdffda93c12a47b2ad553d69fd4e65ea: index
parent_id_2-bdffda93c12a47b2ad553d69fd4e65ea: create
parent_id-bdffda93c12a47b2ad553d69fd4e65ea: create
case ids after update: child_id-bdffda93c12a47b2ad553d69fd4e65ea, parent_id_2-bdffda93c12a47b2ad553d69fd4e65ea, parent_id-bdffda93c12a47b2ad553d69fd4e65ea
dependent case ids after update: 
syncing main_user
case ids before update: child_id-bdffda93c12a47b2ad553d69fd4e65ea, parent_id_2-bdffda93c12a47b2ad553d69fd4e65ea, parent_id-bdffda93c12a47b2ad553d69fd4e65ea
dependent case ids before update: 
child_id-bdffda93c12a47b2ad553d69fd4e65ea: index
case ids after update: child_id-bdffda93c12a47b2ad553d69fd4e65ea, parent_id_2-bdffda93c12a47b2ad553d69fd4e65ea, parent_id-bdffda93c12a47b2ad553d69fd4e65ea
dependent case ids after update: 
<function test_delete_one_of_multiple_indices at 0x937ec80> failed with the following settings:
settings.TESTS_SHOULD_USE_CLEAN_RESTORE = True
settings.TESTS_SHOULD_TRACK_CLEANLINESS = False
F
======================================================================
FAIL: test_delete_one_of_multiple_indices (casexml.apps.phone.tests.test_sync_mode.SyncTokenUpdateTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 45, in inner
    return helper(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 32, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/tzmigration/test_utils.py", line 28, in call_with_settings
    fn(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 28, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 332, in test_delete_one_of_multiple_indices
    child_id: [parent_ref_2]})
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 108, in _testUpdate
    'case {} has unexpected indices'.format(case_id))
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 92, in _checkLists
    self.assertEqual(set(l1), set(l2), msg)
AssertionError: Items in the second set but not the first:
u'parent_id-bdffda93c12a47b2ad553d69fd4e65ea' : case child_id-bdffda93c12a47b2ad553d69fd4e65ea has unexpected indices
```

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py, line 233 - E501 line too long (117 > 115 characters)
* corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py, line 234 - E501 line too long (120 > 115 characters)
* corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py, line 262 - E501 line too long (161 > 115 characters)
* corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py, line 263 - E501 line too long (136 > 115 characters)
* corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py, line 264 - E501 line too long (162 > 115 characters)

File: corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import json
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/tests/test_index_tree.py]: F401 'json' imported but unused

**Comment by czue** [comment]
closing for https://github.com/dimagi/commcare-hq/pull/7543. good riddance.

**Pull request closed without merging**